website/docs/concurrency_control.md

```diff
@@ -71,16 +71,26 @@ hoodie.write.lock.hivemetastore.table
 
 **`Amazon DynamoDB`** based lock provider
 
-Amazon DynamoDB based lock provides a simple way to support multi writing across different clusters
+Amazon DynamoDB based lock provides a simple way to support multi writing across different clusters.  You can refer to the
+[DynamoDB based Locks Configurations](https://hudi.apache.org/docs/configurations#DynamoDB-based-Locks-Configurations)
+section for the details of each related configuration knob.
 
 ```
 hoodie.write.lock.provider=org.apache.hudi.aws.transaction.lock.DynamoDBBasedLockProvider
-hoodie.write.lock.dynamodb.table
-hoodie.write.lock.dynamodb.partition_key
-hoodie.write.lock.dynamodb.region
-hoodie.write.lock.dynamodb.endpoint_url
-hoodie.write.lock.dynamodb.billing_mode
-```
+hoodie.write.lock.dynamodb.table (required)
+hoodie.write.lock.dynamodb.partition_key (optional)
+hoodie.write.lock.dynamodb.region (optional)
+hoodie.write.lock.dynamodb.endpoint_url (optional)
+hoodie.write.lock.dynamodb.billing_mode (optional)
+```
+
+When using the DynamoDB-based lock provider, the name of the DynamoDB table acting as the lock table for Hudi is
+specified by the config `hoodie.write.lock.dynamodb.table`. This DynamoDB table is automatically created by Hudi, so you
+don't have to create the table yourself. If you want to use an existing DynamoDB table, make sure that the `key` column
+is present in the table, used as the partition key.  The config `hoodie.write.lock.dynamodb.partition_key` specifies the
```

Comment:
may be we can remove `used as the partition key` as you explain in the next sentence and partition key here has a different meaning than what we use it for generally ?

Reply:
Basically, I want to point out that the `key` column should be the partition key of the DynamoDB table when user creates it manually.  I rephrased it to be clear.